charts/skypilot/values.yaml

```diff
@@ -343,6 +343,10 @@ ingress:
     # @schema type: [string, number, null]
     cookie-expire: null  # cookie expiration time (should match refresh token lifespan)
 
+  # TLS settings to enable https ingress
```

Comment:
Can we elaborate a bit in the comment for how to set the TLS secret, i.e. people should set the `<release-name>-tls-secrets` manually? (might be good to make that secret name customizable)

Also, could we add explanations in the docs as well? https://docs.skypilot.co/en/latest/reference/api-server/helm-values-spec.html